content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -167,6 +167,19 @@ kubelet resets the restart backoff timer for that container.
 [Sidecar containers and Pod lifecycle](/docs/concepts/workloads/pods/sidecar-containers/#sidecar-containers-and-pod-lifecycle)
 explains the behaviour of `init containers` when specify `restartpolicy` field on it.
 
+### CrashLoopBackOff
+
+`CrashLoopBackOff` is a status that Kubernetes sets for a Pod when a container in the Pod fails to start properly and then continually tries and fails in a loop. This status is a clear indication that Kubernetes is attempting to a container, but the container keeps crashing during startup.
```

Comment:
Does this term show up (`CrashLoopBackOff`) show up in the Kubernetes API?

Reply:
I think `CrashLoopBackOff` is a term from ``kubectl` (we should definitely still document it). It's not an actual Pod status.